### PR TITLE
update README

### PR DESCRIPTION
**Pull request by stevemart**
* general clean up

### CODE REVIEW

**Comment by stevemart** [comment]
to reviewers: the `push` job is using an old master branch that wasn't passing CI, please disregard that result.

File: README.md [hunk 1]
```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-voice-bot.git)
+
+When the DevOps Toolchain Pipeline appears, click on the `Deploy` button to start the deployment. You can optionally, click on the `Delivery Pipeline` to watch the logs as the app is deployed.
+
+![](doc/source/images/toolchain-pipeline.png)
 
-   * wvb-speechtotext
-   * wvb-texttospeech
-   * wvb-assistant
+Once deployed, the app can be viewed by clicking `View app`. The app can also be viewed in the dashboard. The app should be prefixed with the string `watson-voice-bot-`, and the corresponding services that were created and easily identified by the `wvb-` prefix, i.e.: `wvb-assistant`.
```
Comment (rhagarty) [inline README.md]: and the corresponding services that were created can easily be ...

File: README.md [hunk 2]
```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
...
 
-4. Import the Watson Assistant workspace. Follow Step [Import the Conversation workspace](#3-import-the-conversation-workspace) under the Section **Run Locally**
+### 3. Upload the Watson Assistant workspace
 
-   ![](doc/source/images/application_name.png)
+Now that our services are created and the app is deployed we need to update tell the application to use a specific Watson Assistant dialog. We'll be using the file [`data/workspace.json`](data/workspace.json), which documents our entire conversation dialog. To do this, launch the Watson Assistant tool and use the `import` icon button on the right. Find the [`data/workspace.json`](data/workspace.json) file from the cloned repo and import that to the Watson Assistant tool.
```
Comment (rhagarty) [inline README.md]: the app is deployed we need to update the application ...

File: README.md [hunk 3]
```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
...
 
-5. Configure Environment Variable. Open the app starting with the name `CodePattern_VoiceBot` from under the [IBM Cloud Dashboard](https://console.bluemix.net/dashboard/apps/) by clicking on the Application name.
+Each workspace in Watson Assistant has a specific ID, to find the `Workspace ID` for a given workspace, click the context menu of the workspace and select `View details`. The workspace ID can be copied and saved as we'll need it in the next step.
```
Comment (rhagarty) [inline README.md]: Think this needs a period in place of the first comma.

File: README.md [hunk 4]
```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
...
 
-  Click on `Runtime` on the left nav.
+Optionally, to view the conversation dialog select the workspace and choose the **Dialog** tab, here's a snippet of the dialog:
```
Comment (rhagarty) [inline README.md]: Again, period instead of comma at the end

File: README.md [hunk 5]
```diff
@@ -105,39 +121,28 @@ Create the following services:
 * [**Watson Speech To Text**](https://console.bluemix.net/catalog/services/speech-to-text)
 * [**Watson Text To Speech**](https://console.bluemix.net/catalog/services/text-to-speech)
 
-### 3. Import the Conversation workspace
+### 3. Upload the Watson Assistant workspace
 
-Launch the `Watson Assistant` tool. Use the `import` icon button on the right
+Now that our services are created and the app is deployed we need to update tell the application to use a specific Watson Assistant dialog. We'll be using the file [`data/workspace.json`](data/workspace.json), which documents our entire conversation dialog. To do this, launch the Watson Assistant tool and use the `import` icon button on the right. Find the [`data/workspace.json`](data/workspace.json) file from the cloned repo and import that to the Watson Assistant tool.
```
Comment (rhagarty) [inline README.md]: the app is deployed we need to update the application ...

File: README.md [hunk 6]
```diff
@@ -105,39 +121,28 @@ Create the following services:
...
 
-Find the local version of [`data/assistant/workspaces/voicebot.json`](data/assistant/workspaces/voicebot.json) and select
-`Import`. Find the `Workspace ID` by clicking on the context menu of the new
-workspace and select `View details`. Save this ID for later.
+Each workspace in Watson Assistant has a specific ID, to find the `Workspace ID` for a given workspace, click the context menu of the workspace and select `View details`. The workspace ID can be copied and saved as we'll need it in the next step.
 
-*Optionally*, to view the conversation dialog select the workspace and choose the
-**Dialog** tab, here's a snippet of the dialog:
+Optionally, to view the conversation dialog select the workspace and choose the **Dialog** tab, here's a snippet of the dialog:
```
Comment (rhagarty) [inline README.md]: period instead of comma

File: README.md [hunk 7]
```diff
@@ -105,39 +121,28 @@ Create the following services:
...
-### 4. Configure credentials
-
-The credentials for IBM Cloud services (Assistant, Speech To Text and Text To Speech), can be found in the ``Services`` menu in IBM Cloud,
-by selecting the ``Service Credentials`` option for each service.
-
-The other settings for Conversation was collected during the
-earlier setup step (``WORKSPACE_ID``).
-
-Copy the [`env.sample`](env.sample) to `.env`.
+Our services are created and workspace uploaded. It's now time to let our application run locally and to do we'll configure a simple text file with the values we want to use. We begin by copying the the [`env.sample`](env.sample) file and naming it `.env`.
```
Comment (rhagarty) [inline README.md]: let our application run locally and to do that we'll ...

File: README.md [hunk 8]
```diff
@@ -105,39 +121,28 @@ Create the following services:
...
-$ cp env.sample .env (optional)
+cp env.sample .env
 ```
 
-> Note: This is an optional step that you will have to execute in the event where you see that the
-credentials are not getting picked up from the Cloud Runtime (`VCAP_SERVICES`) for some reason.
-Edit the `.env` file with the necessary settings.
+We now populate the key-value pairs with credentials for each IBM Cloud service (Assistant, Speech To Text, and Text To Speech). These values can be found in the `Services` menu in IBM Cloud, by selecting the `Service Credentials` option for each service.
+
+Lastly, the `WORKSPACEID` value was retrieved in the previous step, we use that value here.
```
Comment (rhagarty) [inline README.md]: This needs to be re-worded

File: README.md (hunk 1 above)
Comment (stevemart) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (stevemart) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (stevemart) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (stevemart) [reply README.md]: but i love run on sentences so much.

done.

File: README.md (hunk 5 above)
Comment (stevemart) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (stevemart) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (stevemart) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (stevemart) [reply README.md]: removed the last line, thought i had already removed it.

**Comment by stevemart** [comment]
see new commit in a minute.

File: README.md [hunk 9]
```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+```
 
-1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy``.
+We’ll be using the file [`data/workspace.json`](data/workspace.json).
 
-2. In Toolchains, click on `Delivery Pipeline` to watch while the app is deployed. Once deployed, the app can be viewed by clicking `View app`.
+### 2. Create the services and deploy the web app
 
-   ![](doc/source/images/toolchain-pipeline.png)
+The next step is to deploy the application, we'll do this by running clicking the button below. A nice byproduct of doing doing this is that the services required (Speech-to-Text, Text-to-Speech, and Assistant) will be created automatically. Click the button below.
```
Comment (rhagarty) [inline README.md]: - remove 'running'
- duplicate 'doing'

Comment (stevemart) [reply README.md]: done

File: README.md [hunk 10]
```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-voice-bot.git)
+
+When the DevOps Toolchain Pipeline appears, click on the `Deploy` button to start the deployment. You can optionally, click on the `Delivery Pipeline` to watch the logs as the app is deployed.
+
+![](doc/source/images/toolchain-pipeline.png)
 
-   * wvb-speechtotext
-   * wvb-texttospeech
-   * wvb-assistant
+Once deployed, the app can be viewed by clicking `View app`. The app can also be viewed in the dashboard. The app should be prefixed with the string `watson-voice-bot-`, and the corresponding services that were created and easily be identified by the `wvb-` prefix, i.e.: `wvb-assistant`.
```
[the remark this answers is not in the document]
Comment (stevemart) [reply README.md]: done

File: README.md [hunk 11]
```diff
@@ -105,39 +121,27 @@ Create the following services:
...
-### 3. Import the Conversation workspace
+### 3. Upload the Watson Assistant workspace
 
-Launch the `Watson Assistant` tool. Use the `import` icon button on the right
+Now that our services are created and the app is deployed we need to update the application to use a specific Watson Assistant dialog. We'll be using the file [`data/workspace.json`](data/workspace.json), which documents our entire conversation dialog. To do this, launch the Watson Assistant tool and use the `import` icon button on the right. Find the [`data/workspace.json`](data/workspace.json) file from the cloned repo and import that to the Watson Assistant tool.
 
-Find the local version of [`data/assistant/workspaces/voicebot.json`](data/assistant/workspaces/voicebot.json) and select
-`Import`. Find the `Workspace ID` by clicking on the context menu of the new
-workspace and select `View details`. Save this ID for later.
+Each workspace in Watson Assistant has a specific ID, to find the `Workspace ID` for a given workspace, click the context menu of the workspace and select `View details`. The workspace ID can be copied and saved as we'll need it in the next step.
```
[the remark this answers is not in the document]
Comment (stevemart) [reply README.md]: done

**Comment by stevemart** [comment]
@rhagarty check new commit please

**Pull request merged**